packages/components/src/VirtualizedList/utils/tablerow.js

```diff
@@ -24,8 +26,15 @@ export function insertSelectionConfiguration({ children, isSelected, selectionTo
 				flexShrink={0}
 				flexGrow={0}
 				cellDataGetter={({ rowData }) => isSelected(rowData)}
-				columnData={{ label: 'Select this element', onChange: selectionToggle }}
+				columnData={{
+					label: 'Select this element',
```

Comment:
No translation needed ?

Reply:
I didn't add this, just reformat. Let's add translation in another PR.